### ISSUE TITLE
Pedal Pots

### ISSUE THREAD

**Issue by superami-code** [akakikuumeri/pipechair-cockpit#2]
I have the assembly coming along nicely, I had just enough rest pipe to get things done. When it comes to wiring up the poteniometers for the pedals, I was trying to figure out the wiring.  I have prototyped a solution with just the two 10k Pots, but I am getting a maximum variance of 3.5-4.5v with a 5v source. I was hoping for a little more range. In the video, only the one side is visible and its clear to see you've added at least one resistor to the circuit. Do you have a schematic? Are you measuring each POT independently?

**Comment by akakikuumeri** [comment]
Hi, I added a schematic. I have wired the pots in a differential configuration and I'm reading their single, combined output with one analog pin on the Arduino.
The resistors I have added do not increase the voltage range, they are just to protect from an unlikely short circuit. I also get only a limited voltage range from my pedals, but it is more than enough resolution for the yaw axis. It is expanded to the full 0-100% output by the mmjoy2 firmware.
